### PR TITLE
Advanced filtering changes - not including modals.

### PR DESCRIPTION
**Pull request by martinkimmings**

### CODE REVIEW

**Review by imranbashir01 — DISMISSED** [review]

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Views/Shared/_SolutionSearchDisplay.cshtml [hunk 1]
```diff
@@ -66,39 +55,23 @@
...
-            <h4 class="nhsuk-heading-xs">Search information</h4>
-            <p>
-                @(CapabilitiesNotMet())
-            </p>
+            <h4 class="nhsuk-heading-xs">@FrameworkHeading()</h4>
+            <p>@(string.Join(", ", Model.CatalogueItem.Solution.FrameworkSolutions.Select(fs => $"{fs.Framework.ShortName}{(fs.Framework.IsExpired ? " (expired)" : string.Empty)}")))</p>
+        }
+        else
+        {
+            foreach (var fs in Model.CatalogueItem.Solution.FrameworkSolutions) //.Select(fs => $"{fs.Framework.ShortName}{(fs.Framework.IsExpired ? " (expired)" : string.Empty)}")))</p>
```
Comment (ghost) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Views/Shared/_SolutionSearchDisplay.cshtml]: Commented out code here could be removed

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Views/Shared/_AdditionalFilters.cshtml [hunk 2]
```diff
@@ -2,101 +2,147 @@
...
-
-         <nhs-fieldset-form-label asp-for="@Model">
-            <nhs-checkbox-container>
-				@for (int clientAppTypeIndex = 0; clientAppTypeIndex < Model.ApplicationTypeOptions.Count; clientAppTypeIndex++)
-				{
+        <nhs-fieldset label-text="Quick filters" size="ExtraSmall">
+            <a asp-area="@typeof(SolutionsController).AreaName()"
+               asp-controller="@typeof(SolutionsController).ControllerName()"
+               asp-action="@nameof(SolutionsController.Index)"
+               asp-route-selected="5|11|12|13|14|15">
```
Comment (ghost) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Views/Shared/_AdditionalFilters.cshtml]: Magic numbers. Might be worth creating a consts class, just to give the IDs some meaning in terms of which capabilities they refer to, appending to an array and then doing a `string.Join`

**Review by bjss-colin — CHANGES_REQUESTED** [review]
Two minor changes

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Views/Shared/_SolutionSearchDisplay.cshtml (hunk 1 above)
Comment (martinkimmings) [reply src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Views/Shared/_SolutionSearchDisplay.cshtml]: Good spot.

**Review by bjss-colin — APPROVED** [review]

**Review by imranbashir01 — APPROVED** [review]

**Pull request merged**